packages/generator/src/entity/namespace.ts

```diff
@@ -44,28 +46,64 @@ function properties(entity: VdmEntity): VariableStatementStructure[] {
   return entity.properties.map(prop => property(prop, entity));
 }
 
-function property(
+function getPropertyFieldClassName(prop: VdmProperty): string {
+  return prop.isMulti ? 'CollectionField' : prop.fieldType;
+}
+
+function getPropertyFieldInitializer(
```

Comment:
Is there a property not related to a field? Is getFieldClassName not enough as a name? Also for the other methods.

Reply:
done